metricbeat/etc/beat.full.yml

```diff
@@ -153,10 +153,20 @@ metricbeat.modules:
   #period: 10s
 
   # The host must be passed as PostgreSQL DSN. Example:
-  # postgres://pqgotest:password@localhost:5432?sslmode=disable
+  # postgres://localhost:5432?sslmode=disable
   # The available parameters are documented here:
   # https://godoc.org/github.com/lib/pq#hdr-Connection_String_Parameters
-  #hosts: ["postgres://postgres@localhost:5432"]
+  #
+  # Warning: specifying the user/password in the hosts array is possible
```

Comment:
You missed it in the mongo PR then :-)

I worry that people will still add the user/pass to the connection string because that's how they are used from other software, and then be surprised to see we've indexed the password. Maybe I'm overreacting here, happy to also take it out. But I guess once we do the sanitization part we take out the warnings and we're done with it.